app/controllers/api/service_dialogs_controller.rb

```diff
@@ -31,7 +32,7 @@ def edit_resource(type, id, data)
       rescue => err
         raise BadRequestError, "Failed to update service dialog - #{err}"
       end
-      service_dialog
+      fetch_service_dialogs_content(service_dialog).first
```

Comment:
why is there a .first here and not above on line 22 ?

Reply:
@abellotti it returns an array, and it needs to return the first element which is both the service dialog and the content